docs/en/config_mc/mc_trajectory_tuning.md

```diff
@@ -68,9 +63,6 @@ The following list provides an _overview_ of the different implementations of ho
 
 Analogously to [position mode implementations](#position-mode-implementations) these are the implementations for interpreting vertical stick input:
 
-- [Jerk-limited](../config_mc/mc_jerk_limited_type_trajectory.md)
```

Comment:
Altitude is still jerk limited so this should probably stay.

Reply:
okay brought it back